RELEASING.md

```diff
@@ -21,8 +21,50 @@ the release script to actually perform the release and prepare main for the next
 where `<YOUR_KEY_SHORT_ID>` is the short id of your PGP key
 and `<RELEASE_VERSION>` is a release number such as `0.6.0`.
 
-Once the release is completed, use the [Nexus UI](https://s01.oss.sonatype.org/) to close the staging repository, then release it. That will push the Maven artefacts available
-on Maven Central.  The artefacts will take a while to reach all the Maven Central mirrors.
+The script will push the Maven artefacts to a Nexus **staging repository** and it will open a PR that will commit the release to Git.
+
+The next steps are:
+1.  Validate the staged artefacts.
+2.  If the staging artefacts are **good**, release the stage repository.  If they are not, **drop** the stage repository and start the release over.
+3.  Merge the PR.
+
+These are described below.
+
+### Validating the staged artefacts
```

Comment:
We could have a Git Action for this bit.

Reply:
Yeah that would be very cool, but for another day.